content/courses/tokens-and-nfts/token-program.md

```diff
@@ -707,14 +725,17 @@ Open the token account in Solana Explorer. Look at the owner - it's the account
 you made the ATA for! The balance will be zero, as we haven't sent any tokens
 there yet. Let's mint some tokens there and fix that!
 
+Remember the address of your token account ! We'll use it to mint tokens.
+
 #### Mint Tokens
 
 Now that we have a token mint and a token account, let's mint tokens to the
 token account. Recall that we set the `user` as the `mintAuthority` for the
 `mint` we created.
 
-Create a function `mintTokens` that uses the `spl-token` function `mintTo` to
-mint tokens:
+Create an empty file called `mint-tokens.ts`. Then uses the `spl-token` function
+`mintTo` to mint tokens. Remember to substitute in your token mint address and
```

Comment:
```suggestion
`mintTo()` to mint tokens. Remember to substitute in your token mint address and
```

Reply:
Could you please confirm whether you are suggesting that we add "()" to all function references?

Reply:
It's a bit of convention in programming docs but no, it's not mandatory according to our current guidelines.